benchopt/cli/helpers.py

```diff
@@ -1,6 +1,7 @@
 import click
 import pprint
 from pathlib import Path
+import os
```

Comment:
nitpick: If you could put this import at the top.

Reply:
Fixed in 96ae1b2